app/controllers/scim_rails/scim_users_controller.rb

```diff
@@ -126,9 +126,19 @@ def put_active_param
     end
 
     def patch_active_param
-      active = params.dig("Operations", 0, "value", "active")
-      raise ScimRails::ExceptionHandler::UnsupportedPatchRequest if active.nil?
-      active
+      handle_invalid = lambda do
+        raise ScimRails::ExceptionHandler::UnsupportedPatchRequest
+      end
+
+      valid_operation = params["Operations"].find(handle_invalid) do |operation|
```

Comment:
The argument supplied to "find" is called when no match is found. So if no valid operation is found, it calls the lambda raising the exception.